.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,89 @@
+name: Publish the latest release on PyPI
+
+on:
```

Comment:
lets trigger on pr merge on release.

```
on:
  pull_request:
    types: [closed]
    branches:
      - main
  ```

and there is also stable and not stable release

Reply:
Yeah I considered this, setting it to trigger on PRs that have a tag is more flexible and will be the future workflow. Setting it to main is more of centralized control for now.